doc/policies/admin.rst

```diff
@@ -44,6 +44,20 @@ to enable tokens in the user-realm *sales*.
 The following actions are available in the scope
 *admin*:
 
+tokenlist
+~~~~~~~~~
+
+type: bool
+
+This allows the administrator to list existing tokens in the specified user realm.
+Note, that the resolver in this policy is ignored.
+
+If the policy with the action ``tokenlist`` is not bound to any user realm, this acts
+as a wild card and the admin is allowed to list all tokens.
+
+If the action ``tokenlist`` is not active, but admin policies exist, then the admin
```

Comment:
Should we add a sentence what happens if multiple policies with a ``tokenlist`` action are defined? I think in this case, priorities are ignored and the admin is allowed to list tokens in all mentioned realms.

Reply:
I think this is as with all simple bool policies. Yes, I can add something.

Reply:
done in 80747ff6